### PR TITLE
Add the `problem solution tab` to the creator

### PR DESCRIPTION
**Pull request by Aritra8438**
# Description
Add solutions tab to the problem creator and adds test for it.

Fixes: #7595 

https://github.com/omegaup/omegaup/assets/64671908/9bfaa54b-ee1c-4198-ac38-3351eba33565

# Comments

- [x] Add a similar (to statement tab) component for the solution tab.
- [x] Add the same tests for it.

# Checklist:

- [x] The code follows the [coding guidelines](https://github.com/omegaup/omegaup/wiki/Coding-guidelines) of omegaUp.
- [x] The tests were executed and all of them passed.
- [x] If you are creating a feature, the new tests were added.
- [x] If the change is large (> 200 lines), this PR was split into various Pull Requests. It's preferred to create one PR for changes in controllers + unit tests in PHPUnit,  and then another Pull Request for UI + tests in Jest, Cypress or both.

### CODE REVIEW

**Comment by Aritra8438** [comment]
Hi @carlosabcs, PTAL!

File: frontend/templates/en.lang [hunk 1]
```diff
@@ -1038,6 +1038,7 @@ problemCreatorNumberOfCasesHelper = "The cases will have the name:"
 problemCreatorPoints = "Points"
 problemCreatorPrefix = "Prefix"
 problemCreatorSolution = "Solution"
+problemCreatorSolutionMarkdownSave = "Save"
```
Comment (carlosabcs) [inline frontend/templates/en.lang:1041]: You can reuse the `problemreatorMarkdownSave` translation string.

**Review by carlosabcs — APPROVED** [review]
Just remove the unnecessary string and that's all.

File: frontend/templates/en.lang (hunk 1 above)
Comment (Aritra8438) [reply frontend/templates/en.lang:1041]: Done!

**Comment by Aritra8438** [comment]
@heduenas, PTAL!

**Pull request merged**